src/js/components/RadioButton/RadioButton.js

```diff
@@ -41,8 +41,20 @@ const RadioButton = forwardRef(
 
     const Icon = theme.radioButton.icons.circle;
     let borderColor = normalizeColor(theme.radioButton.border.color, theme);
+    let backgroundColor = normalizeColor(
+      theme.radioButton.background.color,
+      theme,
+    );
+
     if (checked) {
       borderColor = normalizeColor(theme.radioButton.color || 'control', theme);
+
+      if (theme.radioButton.check.background.color) {
+        backgroundColor = normalizeColor(
+          theme.radioButton.check.background.color || 'light-1',
```

Comment:
In case `theme.radioButton.check.background.color` is undefined, why are you assuming we should default to `light-1` instead of not assigning anything and keeping `backgroundColor` as is?

Reply:
I can't remember exactly what was my thought process to that, but I was pretty sure that it was related to the default `theme.radioButton.background.color` on the base theme. I see my mistake now :sweat_smile:

Reply:
So sorry for taking that long. Had to deal with another things.
Anyway, resolved in 5700ce5